package.json

```diff
@@ -75,12 +75,12 @@
     "min-wd": "^2.11.0",
     "mocaccino": "^4.1.0",
     "mocha": "^5.2.0",
-    "puppeteer": "^1.10.0",
+    "puppeteer": "^1.13.0",
```

Comment:
Looking at the changelog for this: https://github.com/GoogleChrome/puppeteer/releases I've seen that puppeteer now has experimental Firefox support. Is this something mochify should/could be supporting too?

Reply:
Absolutely!

Reply:
I just tried hot-swapping and it seems like it's not entirely working like the Chrome version as `page.on('console'` will never be called (and all `file://` based tests time out) which looks to be caused by this issue: https://github.com/GoogleChrome/puppeteer/issues/3652

I guess I'll just open an issue for adding the option once the upstream bug has been resolved.